### ISSUE TITLE
Squiz.Strings.EchoedStrings does not properly fix bracketed statements

### ISSUE THREAD

**Issue by u01jmg3** [squizlabs/PHP_CodeSniffer#1150]
Using the following rules and PHP, PHPCBF makes an error during correction.

```xml
<rule ref="Squiz.Strings.EchoedStrings" />
<rule ref="Squiz.WhiteSpace.SemicolonSpacing" />
```

```php
echo($y) ;

// becomes...

echo $y);
```

--

Modifying [`EchoedStringsSniff.php#L67`](https://github.com/squizlabs/PHP_CodeSniffer/blob/e11f25393c3e5795ff87f2ccb2c156c63d6a13b3/CodeSniffer/Standards/Squiz/Sniffs/Strings/EchoedStringsSniff.php#L67) and including `T_WHITESPACE` solves the problem.

```php
// with code extended

echo $y;
```

**Comment by gsherwood** [comment]
Debug output from --report=diff -vvv:
```
*** START FILE FIXING ***
---START FILE CONTENT---
1|<?php
2|echo($y) ;
3|
--- END FILE CONTENT ---
       	E: [Line 2] Echoed strings should not be bracketed (Squiz.Strings.EchoedStrings.HasBracket)
       	=> Changeset started by Squiz_Sniffs_Strings_EchoedStringsSniff (line 89)
       		Q: Squiz_Sniffs_Strings_EchoedStringsSniff (line 90) replaced token 2 (T_OPEN_PARENTHESIS) "(" => ""
       		Q: Squiz_Sniffs_Strings_EchoedStringsSniff (line 91) replaced token 5 (T_WHITESPACE) "·;" => ";"
       		A: Squiz_Sniffs_Strings_EchoedStringsSniff (line 92) replaced token 2 (T_OPEN_PARENTHESIS) "(" => ""
       		A: Squiz_Sniffs_Strings_EchoedStringsSniff (line 92) replaced token 5 (T_WHITESPACE) "·;" => ";"
       	=> Changeset ended: 2 changes applied
       	E: [Line 2] Space found before semicolon; expected ");" but found ") ;" (Squiz.WhiteSpace.SemicolonSpacing.Incorrect)
       	=> Changeset started by Squiz_Sniffs_WhiteSpace_SemicolonSpacingSniff (line 90)
       		Q: Squiz_Sniffs_WhiteSpace_SemicolonSpacingSniff (line 92) replaced token 5 (T_WHITESPACE) ";" => ";"
       		* token 5 has already been modified, skipping *
       	=> Changeset failed to apply
       	* fixed 2 violations, starting loop 2 *
---START FILE CONTENT---
1|<?php
2|echo$y);
3|
--- END FILE CONTENT ---
       	E: [Line 2] Language constructs must be followed by a single space; expected "echo $y" but found "echo$y" (Squiz.WhiteSpace.LanguageConstructSpacing.Incorrect)
       	Squiz_Sniffs_WhiteSpace_LanguageConstructSpacingSniff (line 93) replaced token 1 (T_ECHO) "echo" => "echo·"
       	* fixed 1 violations, starting loop 3 *
---START FILE CONTENT---
1|<?php
2|echo $y);
3|
--- END FILE CONTENT ---
*** END FILE FIXING ***
```

**Comment by gsherwood** [comment]
Thanks for the bug report. I found another issue with the fixer that I also corrected (open brace wasn't given a space before it when needed).

**Comment by u01jmg3** [comment]
Will this be included in `v3.0.1`?

**Comment by gsherwood** [comment]
> Will this be included in v3.0.1?

3.0.1 is not the next 3.x release (3.0.0 isn't out yet), but it will be in both the next 2.x and 3.x releases.
